pkg/virt-controller/watch/clone/clone_test.go

```diff
@@ -101,14 +100,20 @@ var _ = Describe("Clone", func() {
 	}
 
 	addClone := func(vmClone *clonev1alpha1.VirtualMachineClone) {
+		var err error
+		vmClone, err = client.CloneV1alpha1().VirtualMachineClones(testNamespace).Update(context.TODO(), vmClone, metav1.UpdateOptions{})
```

Comment:
The `Update` was made because in the global `BeforeEach` the vmClone resource was already added:
```
client = kubevirtfake.NewSimpleClientset(vmClone)
```
I converted the latter in:
```
client = kubevirtfake.NewSimpleClientset()
```
and switched using `Create`. WDYT?

Reply:
I like it more.